index/src/executors/bridge_executors/moonbeam_to_phala.rs

```diff
@@ -40,11 +41,13 @@ impl BridgeExecutor for Moonbeam2PhalaExecutor {
         asset_contract_address: Vec<u8>,
         recipient: Vec<u8>,
         amount: u128,
-    ) -> core::result::Result<(), Error> {
+        extra: ExtraParam,
+    ) -> core::result::Result<Vec<u8>, Error> {
         let signer = KeyPair::from(signer);
         let token_address = Address::from_slice(&asset_contract_address);
+
         // TODO: better error handling
-        _ = self
+        let tx_id = self
```

Comment:
Is it possible to merge `moon_to_acala` and `moon_to_phala` executors?

Reply:
I think we can, they both use `XtokenClient` and with only paraid are different. @tenheadedlion please do it in a future PR

Reply:
will do